src/Symfony/Component/PropertyInfo/PropertyInfo.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyInfo;
+
+/**
+ * Default {@see PropertyInfoInterface} implementation.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class PropertyInfo implements PropertyInfoInterface
+{
+    /**
+     * @var PropertyListRetrieverInterface[]
+     */
+    private $listExtractors;
```

Comment:
i mean this:
```php
/**
 * @var PropertyListRetrieverInterface[]
 */
private $listExtractors;

/**
 * @var PropertyListRetrieverInterface[]
 */
private $listExtractors;
```

instead of this:
```php
/**
 * @var PropertyListRetrieverInterface[]
 */
private $listExtractors;
/**
 * @var PropertyListRetrieverInterface[]
 */
private $listExtractors;
```

maybe you missunderstood me or i missunderstood you :smile: